website_version_ce/controllers/main.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-#
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import datetime
+from openerp import http
+from openerp.http import request
+from openerp.addons.website.controllers.main import Website
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+GOOGLE_ANALYTICS_CONFIGURED = 2
+GOOGLE_ANALYTICS_PARTIALLY_CONFIGURED = 1
+GOOGLE_ANALYTICS_NOT_CONFIGURED = 0
+
+
+class VersioningController(Website):
+
+    @http.route('/website_version_ce/change_version', type='json', auth="user",
```

Comment:
I think it's to avoid conflict with the enterprise website_version, but maybe website_version_ce is not the best name. Any suggestion ?